rules-reviewed/hibernate/hibernate.windup.xml

```diff
@@ -22,7 +22,7 @@
             <javaclass references="org.hibernate.annotations.Type" matchesSource="{*}text{*}" />
          </when>
          <perform>
-            <hint title="@Type(type=text) is deprecated in Hibernate 4" effort="1" category-id="mandatory">
+            <hint title="Hibernate 4 - Deprecated @Type(type=text)" effort="1" category-id="optional">
```

Comment:
The rewording of titles to take the last word and move it to the start is not really significant - this is not related to only this title change as here is a category-id change too, but wherever is only this rewording that is the matter of taste - not really additional value ;)